### PR TITLE
setup basic classes / models

### PR DESCRIPTION
**Pull request by bvcage**
Moved Blog Post Type & User Role to separate issues. Built basic Blog Post model & moved remaining attributes to separate issue. Remaining acceptance criteria are met - ready for review.

### CODE REVIEW

File: bvc_blog/blog/models.py [hunk 1]
```diff
@@ -1,3 +1,60 @@
...
+
+class SocialMediaPlatform(models.Model):
+    name = models.CharField(max_length=255)
+    icon = models.CharField(max_length=255)
+    description = models.TextField()
+
+class User(models.Model):
+    username = models.CharField(max_length=255)
+    email = models.EmailField()
+    # password =
```
Comment (DosLlamas) [inline bvc_blog/blog/models.py:51]: Finish the password attribute: 
# password = models.CharField(max_length=50)
# or
# password = models.PasswordField(max_length=50)

Comment (DosLlamas) [reply bvc_blog/blog/models.py:51]: Everything else looks great to me

**Review by DosLlamas — APPROVED** [review]
Finish the password attribute for the User model. Other than that, this looks great to me

File: bvc_blog/blog/models.py (hunk 1 above)
Comment (bvcage) [reply bvc_blog/blog/models.py]: Added password attribute

**Review by DosLlamas — COMMENTED** [review]
Perfect

**Pull request merged**